youtube_dl/extractor/wdr.py

```diff
@@ -16,59 +17,45 @@
 
 
 class WDRIE(InfoExtractor):
-    _PLAYER_REGEX = '-(?:video|audio)player(?:_size-[LMS])?'
-    _VALID_URL = r'(?P<url>https?://www\d?\.(?:wdr\d?|funkhauseuropa)\.de/)(?P<id>.+?)(?P<player>%s)?\.html' % _PLAYER_REGEX
```

Comment:
Now it does not handle URLs like `http://www1.wdr.de/mediathek/video/sendungen/hier_und_heute/videostreetfoodpioniere100-videoplayer_size-L.html`. `(?P<player>%s)?\.html' % _PLAYER_REGEX` was on purpose.